certified-connectors/Teams-Spirit/apiProperties.json

```diff
@@ -0,0 +1,53 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "{clientId}",
+          "scopes": [
+            "api://app.teams-spirit.com/3a57026a-cf3f-4295-93f5-ca245af3d2df/access_as_user"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "AzureActiveDirectoryResourceId": "3a57026a-cf3f-4295-93f5-ca245af3d2df",
+            "IsOnbehalfofLoginSupported": true
+          },
+          "customParameters": {
+            "loginUri": {
+              "value": "https://login.windows.net"
+            },
+            "tenantId": {
+              "value": "common"
+            },
+            "resourceUri": {
+              "value": "3a57026a-cf3f-4295-93f5-ca245af3d2df"
```

Comment:
@tillkaemmerer secret look alike values are not allowed. Please replace it with dummy values.

Reply:
@Amjed-Ayoub The ResourceId and ResourceUri are required to be our App-Registration and not the one a user would create in their tenant. I updated them to be in the form of api://app.team-spirit.com/3a57026a-cf3f-4295-93f5-ca245af3d2df therefore they can not be mistaken as secret values.